integr8ly/cloud-resource-operator
# Postgres default Engine upgrade

## Overview

This is WIP and needs some cleanup and test updates

## Verification

- Check out master
- Run `make cluster/prepare`
- Run `make run`
- Run `make cluster/seed/managed/postgres`
- verify postgres is engine version 1.15 and that no service update is showing in the UI
- Checkout this branch
- Rerun the code
- Add applyImmediately true to the postgres cr to trigger the engine upgrade
- Once the engine upgrade is present see that the system update is available
- the rest is todo but the idea is that the service update gets applied immediately after the engine upgrade.


## Checklist
